src/amo/components/MetadataCard/index.js

```diff
@@ -31,10 +31,15 @@ const MetadataCard = ({ className, metadata }: Props): React.Node => {
         return (
           // eslint-disable-next-line react/no-array-index-key
           <dl className="MetadataCard-list" key={index}>
-            <dd className="MetadataCard-content">
+            <dd
+              aria-label={hasContent ? content : 'Loading'}
```

Comment:
I'm really not sure about this use - https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label says this property is for accessibility when there _isn't_ any appropriate text content to reference what the element is. Here we're just repeating the exact text content.

Reply:
You're right. I found other places where it also says not to do this. According to the docs I've found, the accessible name for `dd` and `dt` *should be* the text content, much like it is for `button` and `link`, but React Testing Library seems to report the `name` as `""` for any `dt` or `dd`. I discovered, however, that using `description` rather than `name` does work, so that's a good solution for us. I have updated this PR to revert the changes to the component, and I have also updated the test to serve as an example of how we can target `dd` and `dt`.